src/main/webapp/index.html

```diff
@@ -37,10 +38,16 @@ <h1>CodeU Starter Project</h1>
 
     <figure>
       <figcaption>Try out Text-To-Speech:</figcaption>
-      <audio controls src="FILL IN WITH SOMETHING">
-        Your browser does not support the
-        <code>audio</code> element.
-      </audio>
+      <audio id="audio-" src=""></audio>
```

Comment:
Does src need anything within the quotes? Just looking at the line it's compared too which says "fill in with something".

Reply:
Should be okay with empty quotes! In the third commit I updated the implementation to have the js fill in the src tag.